### DISCUSSION TITLE
Battery consumption

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#122] [Q&A] [answered]
Cruiser was already optimized in battery usage, however in the latest version:

Cruiser's communication with the device's sensors has been further improved,
so you should expect even less battery consumption!

**Comment by ramondevic** [comment]
Good job :)

**Reply by DolomitesTom** [reply]
Not sure about this. Had problems with accu yesterday, but could have been a too poor power source in an older car. Had to activate display off timer, deactivate overlay gps app and switch to Google location service in order to not lose power. Tomorrow on the bigger tour with motorbike I will test intensively on my own bike where I know the power supply is 3A at the output.

**Comment by devemux86** [comment, marked as answer]
Lower battery consumption is implemented in [Cruiser 3.0.17](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Comment by DolomitesTom** [comment]
All good with the right equipment. It was the old car that obviously didn't deliver enough output, neither on the USB nor on the euro car port.

On the motorbike with 3A fully loaded quickly, also with Android location service and without screen saver.

**Reply by devemux86** [reply]
@DolomitesTom that's great, thanks for the feedback!

**Comment by ToToGSARider** [comment]
Hello everyone, I use a Samsung S20+ on my motorcycle. With the curvy 2 app I have a battery consumption of about 17% in an hour. In the background still runs a hint app and the BMW app that records my tours. Do I achieve the same low energy consumption with the cruiser app? While driving I charge my phone with the SP Connect induction plate about 1% in 10 minutes during active navigation or follow mode.

Translated with www.DeepL.com/Translator (free version)

Hallo zusammen, ich nutze bei mir am Motorrad ein Samsung S20+. Mit der kurviger 2 App habe ich einen Akku Verbrauch von ca. 17% in einer Stunde. Im Hintergrund läuft noch eine Hinweis App und die BMW App die meine Touren aufzeichnet. Erreiche ich mit der Cruiser App den gleichen niedrigen Energieverbrauch? Während der fahrt lade ich mein Telefon mit der SP Connect Induktionsplatte ca. 1% in 10 Minuten während der aktiven Navigation oder im Folgemodus.

**Reply by devemux86** [reply]
Hi @ToToGSARider,

K1 / K2 are based on the Cruiser platform, so you can expect at least the same low battery consumption on the Cruiser as well.
However, Cruiser has advanced a lot and the battery consumption has dropped even more! :slightly_smiling_face: 

Please also check the battery optimization guide with many tips for Cruiser configuration:
- https://github.com/devemux86/cruiser/discussions/139

---

K1 / K2 basieren auf der Cruiser-Plattform, daher können Sie auch beim Cruiser mindestens den gleichen niedrigen Akkuverbrauch erwarten.
Allerdings hat sich Cruiser stark weiterentwickelt und der Akkuverbrauch ist noch weiter gesunken! :slightly_smiling_face: 

Bitte beachten Sie auch die Anleitung zur Batterieoptimierung mit vielen Tipps zur Cruiser-Konfiguration:
- https://github.com/devemux86/cruiser/discussions/139

**Reply by ToToGSARider** [reply]
Hallo,

so ich habe heute die Testversion abonniert. Leider habe ich hier das Ergebnis, dass wenn man die geplante Route verlässt und nicht mehr auf seiner geplanten Route ist, das Batteriesymbol sich rot einfärbt, da der Akku Leistung verliert. Ich habe die Neuberechnung so wie früher bei K1 und K2 deaktiviert. Ich habe auch während des Tests dann die Offlinekarte geladen, da ich gedacht habe, das es von der Onlinekarte gekommen ist. Naja, einen Versuch war es Wert. Das Gerät wird über die SPC Induktionsplatte geladen und ich vermute einfach, dass die Cruiser App mehr Strom benötigt als früher, wenn die geplante Route verläßt. Ich zoome dann mit dem WunderlinQ aus der Karte raus und fahre wieder zur ursprünglichen Route zum Beispiel bei Umleitungen.

Hello,

so I subscribed to the trial version today. Unfortunately, I have the result here that when you leave the planned route and are no longer on your planned route, the battery icon turns red as the battery loses power. I disabled the recalculation as I did earlier with K1 and K2. I have also during the test then loaded the offline map, thinking that it came from the online map. Well, it was worth a try. The device is charged via the SPC induction plate and I just suspect that the Cruiser app needs more power than before when leaving the planned route. I then zoom out of the map with the WunderlinQ and drive back to the original route for example with detours.


Translated with www.DeepL.com/Translator (free version)

**Reply by devemux86** [reply]
Hi @ToToGSARider!

Battery indicator shows a warning when:
- Battery drops below 20%.
- Phone is charged, but battery level is dropping.

What settings do you have enabled in Cruiser?
Can you export the .pref file so we can review it?

Please also see the [Battery optimization guide](https://github.com/devemux86/cruiser/discussions/139) for valuable advice on how to setup Cruiser for optimal battery.

---

Die Batterieanzeige zeigt eine Warnung an, wenn:
- der Akku unter 20% fällt
- Das Telefon ist geladen, aber der Akkustand sinkt

Welche Einstellungen haben Sie in Cruiser aktiviert?
Können Sie die .pref-Datei exportieren, damit wir sie überprüfen können?

Bitte lesen Sie auch den [Leitfaden zur Batterieoptimierung](https://github.com/devemux86/cruiser/discussions/139) mit wertvollen Hinweisen zur optimalen Einstellung von Cruiser für einen optimalen Akku.

**Reply by ToToGSARider** [reply]
Hallo @devemux86 ,

wo muß ich in der App hin um die Einstellungen zu exportieren?

Hello @devemux86 ,

where do I have to go in the app to export the settings?

**Reply by devemux86** [reply]
@ToToGSARider 

It is in "Settings | Application | Advanced | Preferences | Export"
or write "export" in the search in settings.

Sie finden sie unter "Einstellungen | Applikation | Erweitert | Voreinstellungen | Exportieren".
oder geben Sie "export" in die Suche in den Einstellungen ein.

**Reply by ToToGSARider** [reply]
Okay, ist gerade per Mail raus.

Okay, just went out in the mail.

**Reply by devemux86** [reply]
@ToToGSARider 

Thanks for the .pref file. Some optimization notes:
- Disable Kalman filter in location settings
- Try "Google" as service in location settings
- Disable "Snap to route" in navigation advanced settings

Vielen Dank für die .pref-Datei. Einige Hinweise zur Optimierung:
- Deaktivieren Sie den Kalman-Filter in den Standorteinstellungen
- Versuchen Sie "Google" als Dienst in den Standorteinstellungen
- Deaktivieren Sie "Position auf Route" in den erweiterten Navigationseinstellungen

**Reply by ToToGSARider** [reply]
Okay, Kalman war noch an und den Standortservice habe ich auf Google geändert. Aber war Android nicht der Dienst der Stromsparender ist? Das andere war deaktiviert. 
Es gab ja nur Probleme wenn ich die Route verlassen habe.

Okay, Kalman was still on and I changed the location service to Google. But wasn't Android the service that was more power efficient? The other was disabled.
There were only problems when I left the route.

**Reply by devemux86** [reply]
@ToToGSARider 

> Aber war Android nicht der Dienst der Stromsparender ist?

Google advertises that its service is more power efficient, but that also depends on the device sensor.
Google wirbt damit, dass sein Dienst stromsparender ist, aber das hängt auch vom Gerätesensor ab.

> Es gab ja nur Probleme wenn ich die Route verlassen habe.

This is strange, because it should actually work the same way. Was the battery discharged?
Please try with the changed settings (Kalman off, Android or Google) and let us know the result.

Das ist seltsam, denn eigentlich sollte es genauso funktionieren. War die Batterie entladen?
Bitte versuchen Sie es mit den geänderten Einstellungen (Kalman deaktiviert, Android oder Google) und teilen Sie uns das Ergebnis mit.

**Reply by ToToGSARider** [reply]
Naja entladen würde ich jetzt nicht sagen. Sie war zwischen 30 und 40 Prozent. Ich habe mir mal eine kleine Runde zusammen geklickt und werde heute Abend noch einmal testen was passiert wenn ich die Route verlasse. Ich habe bei uns in der Region eine Ecke wo aktuell eine Baustelle ist und dort muß ich einer Umleitung folgen. Ich habe die Offlinekarten für mein Bundesland geladen und habe die Neuberechnung deaktiviert. Es ist alles nur sehr seltsam, da ich bei der K1 den Android Service laufen haben. Kann ich so eine Datei auch aus der K1 an euch senden als Vergleich. Hier läuft alles super. Ich werde noch Wahnsinnig mit den Apps. 

Well, I wouldn't say discharged. It was between 30 and 40 percent. I have clicked me times a small round together and will test tonight again what happens when I leave the route. I have with us in the region a corner where currently a construction site is and there I have to follow a detour. I have loaded the offline maps for my state and have disabled recalculation. It's all just very strange since I have the Android service running on the K1. Can I also send such a file from the K1 to you as a comparison. Everything is running great here. I'm still going crazy with the apps. 

Translated with www.DeepL.com/Translator (free version)

**Reply by devemux86** [reply]
@ToToGSARider 

I meant that the battery indicator turns red when the battery level drops.
e.g. 40% -> 39%

Please also run tests with the Google location service.
Also turn off the Kalman filter.

---

Ich meinte, dass die Batterieanzeige rot wird, wenn der Batteriestand sinkt.
z.B. 40% -> 39%

Bitte führen Sie auch Tests mit dem Google-Standortdienst durch.
Schalten Sie auch den Kalman-Filter aus.

**Reply by ToToGSARider** [reply]
@devemux86 Ja genau das ist passiert. Ich habe den Google Dienst jetzt aktiviert und den Kalmann-Filter deaktiviert und werde später eine Runde fahren und dann Infos senden.

@devemux86 Yes that's exactly what happened. I have now activated the Google service and deactivated the Kalmann filter and will go for a ride later and then send info.

**Reply by ToToGSARider** [reply]
@devemux86 so ich bin zurück von der Testrunde und egal was ich gemacht habe, die Induktionsplatte hat geladen. Ich bin mit ca.39-40% gestartet und hatte dann nach ca. 1,5 Std. 52% mit Zwischenstopp wo ich das Handy mal kurz nicht geladen habe. Ziel erreicht. Danke für die Unterstützung. 

@devemux86 so I'm back from the test round and no matter what I did, the induction plate has charged. I started with ca.39-40% and then had after about 1.5 h 52% with intermediate stop where I have not charged the phone times briefly. Goal achieved. Thanks for the support.

**Reply by devemux86** [reply]
@ToToGSARider 

Thanks for the feedback! Next time you can also try the "Android" location service.
("Kalman filter" may not be suitable for wireless charging depending on the device)

Vielen Dank für die Rückmeldung! Das nächste Mal können Sie auch den "Android"-Ortungsdienst ausprobieren.
("Kalman-Filter" ist je nach Gerät möglicherweise nicht für drahtloses Laden geeignet)

**Reply by ToToGSARider** [reply]
@devemux86 mit dem Android Dienst, hatte ich ja das Problem, dass der Akku dann im Folgemodus wenn ich die Route verlassen habe Leistung verloren hat. Das Laden des Akkus lief mit dieser Einstellung perfekt. Mit dem Android Dienst hat das Handy nur geladen wenn ich die Route nicht verlassen habe. Der Kalman war die ganze Zeit deaktiviert. 

@devemux86 with the Android service, I had the problem that the battery then lost power in follow mode when I left the route. The charging of the battery ran perfectly with this setting. With the Android service, the phone only charged when I did not leave the route. The Kalman was disabled all the time.

**Reply by devemux86** [reply]
@ToToGSARider 

I thought that in your initial report you had enabled both Android location service and Kalman filter?

Ich dachte, dass Sie in Ihrem ersten Bericht sowohl den Android-Standortdienst als auch den Kalman-Filter aktiviert hatten?

**Reply by ToToGSARider** [reply]
@devemux86 vor 3 Tagen hatte ich noch den Android Dienst und den Kalman aktiv. Heute hatte ich alles so eingestellt wie es hier in der Batterie Einstellung vorgeschlagen wird.

@devemux86 3 days ago I still had the Android service and the Kalman active. Today I had everything set as suggested here in the battery setting.

**Comment by ToToGSARider** [comment]
Hallo zusammen, so heute die 1. Große Tour mit der Cruiser App gefahren. Ich bin so glücklich, dass ich wieder die gleiche Qualität habe wie in der K1 und K2. Das Handy hat optimal geladen über die SPC Induktionsplatte und ich habe keinen Strom während der Fahrt verloren. Danke für die ganzen Tipps zur Einstellung des Akkus.

Hi all, so today the 1st Grand Tour with the Cruiser app driven. I am so happy to have the same quality again as in the K1 and K2. The phone has charged optimally via the SPC induction plate and I have not lost power while driving. Thanks for all the tips on setting the battery.

**Reply by devemux86** [reply]
@ToToGSARider thanks for the feedback about your tour with Cruiser!

**Comment by devemux86** [comment]
Optimizing Cruiser navigation for the new features of [nearby favorites](https://github.com/devemux86/cruiser/discussions/209) and [nearby POI](https://github.com/devemux86/cruiser/discussions/9),

I found a big improvement in battery life, especially when we are away from the route.

You can expect this new even lower battery consumption in Cruiser in the next update! :muscle:

**Reply by devemux86** [reply]
Even lower battery consumption in navigation is available in [Cruiser 3.0.25](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
